docs/tutorials/build/sync-table.md

```diff
@@ -0,0 +1,982 @@
+        method: "GET",
+        url: url,
+      });
+      let rows = response.body.results;
+      for (let row of rows) {
+        row.thumbnail = row.formats["image/jpeg"];
+        row.link =
+          "https://www.gutenberg.org/ebooks/" + row.id;
+      }
+      let continuation;
+      if (response.body.next) {
+        continuation = {
+          url: response.body.next
+        };
+      }
+      return {
+        result: rows,
+        continuation: continuation,
+      };
+    },
+    ```
+
+    </div>
+    </section>
+
+=== ":material-numeric-2-circle: Read continuation"
+
+    <section class="tutorial-row" markdown>
+    <div markdown>
+
+    The Pack is now returning the continuation appropriately, but unless you use the value stored within the Pack will keep fetching the first page of results over and over again.
+
+    The continuation object is passed into the formula in `context.sync.continuation`. This value will be `undefined` for the first execution, and populated for all following executions.
+
+    If it's populated, retrieve the URL saved into it at the end of the last execution. Cast the value as a `string` since that's what you stored in it. Use it instead of the URL constructed above to continue where you left off.
+
+    </div>
+    <div markdown>
+
+    ```{.ts hl_lines="7-9"}
+    execute: async function (args, context) {
+      let [topic] = args;
+      let baseUrl = "https://gutendex.com/books";
+      let url = coda.withQueryParams(baseUrl, {
+        topic: topic,
+      });;
+      if (context.sync.continuation) {
+        url = context.sync.continuation!.url as string;
+      }
+      let response = await context.fetcher.fetch({
+        method: "GET",
+        url: url,
+      });
+      let rows = response.body.results;
+      for (let row of rows) {
+        row.thumbnail = row.formats["image/jpeg"];
+        row.link =
+          "https://www.gutenberg.org/ebooks/" + row.id;
+      }
+      let continuation;
+      if (response.body.next) {
+        continuation = {
+          url: response.body.next
+        };
+      }
+      return {
+        result: rows,
+        continuation: continuation,
+      };
+    },
+    ```
+
+    </div>
+    </section>
+
+---
+
+### Try it out
+
+Now that you've added in continuations, see if you can fetch the full set of results.
+
+<section class="tutorial-row" markdown>
+<div markdown>
+
+Rebuild the Pack and then click refresh icon (:material-refresh:) in the sync table.
+
+If everything is working correctly you should have more than 32 rows in your sync table.
+
+The Pack maker tools will show multiple executions of the sync formula, with those after the first marked as continuations.
+
+!!! info "Truncated results"
+    Sync tables have a maximum number of rows they can store, which varies depending on what plan you are subscribed to. When your sync hits that limit the sync formula will be terminated, even if you returned another continuation.
```

Comment:
Nit, maybe "plan" --> "Coda plan" to be clear.

Reply:
Done.